src/styles/utils/spacing.ts

```diff
@@ -119,6 +119,10 @@ export default {
         marginRight: 20,
     },
 
+    mr23: {
+        marginRight: 23,
+    },
+
```

Comment:
Same ^

Reply:
Also, are we sure that shouldn't be 24? We basically always use base4 for all of our numeric values.

Reply:
yup - 24 will not be pixel perfect
this is 24
@shawnborton 
<img width="704" alt="Screenshot 2024-03-14 at 18 51 13" src="https://github.com/Expensify/App/assets/28352309/70844d19-7860-4d81-b117-4661f2117a2f">

Reply:
Seems 24 is in prod as well - but text will be not on the same line. If we ok with that i can switch to 24

Reply:
Hmm I guess I'm just trying to understand how 23 actually works and 24 doesn't... where is this 1px coming from? But, given that this is a one-off button style, I can live with it and hopefully some day we revisit all of our row styles.